src/components/carousel/_index.scss

```diff
@@ -0,0 +1,165 @@
+@use "../../scss";
+@use "~slick-carousel/slick/slick";
+
+.c-carousel {
+	@include scss.component-properties("carousel");
+
+	/* Arrows */
+	.slick-prev,
+	.slick-next {
+		cursor: pointer;
+		font-size: 0;
+		line-height: 0;
+		position: absolute;
+		top: 50%;
+		display: block;
+
+		&:hover {
+			color: transparent;
+			outline: none;
+			background: transparent;
+
+			&:before {
+				@include scss.component-properties("carousel-buttons-hover-before");
+			}
+		}
+
+		&:focus {
+			color: transparent;
+			outline: none;
+			background: transparent;
+
+			&:before {
+				@include scss.component-properties("carousel-buttons-focus-before");
+			}
+		}
+
+		@include scss.component-with-states("carousel-buttons", [hover, focus]);
+	}
+
+	.slick-next {
+		&:before {
+			font-family: "Arial", serif;
+			content: ">";
+			-webkit-font-smoothing: antialiased;
+			-moz-osx-font-smoothing: grayscale;
+		}
+
+		@include scss.component-with-states("carousel-button-next", [before]);
+	}
+
+	.slick-prev {
+		&:before {
```

Comment:
This is a duplicate of `.slick-next:before` we can combine them to be more efficient